core/src/main/resources/application.conf

```diff
@@ -0,0 +1,106 @@
+collector {
+  enableDefaultRedirect = false
+
+  paths {
+
+  }
+
+  crossDomain {
+    enabled = false
+    domains = [ "*" ]
+    secure = true
+  }
+
+  cookieBounce {
+    enabled = false
+    name = "n3pc"
+    fallbackNetworkUserId = "00000000-0000-4000-A000-000000000000"
+  }
+
+  cookie {
+    enabled = true
+    expiration = 365 days
+    name = sp
+    domains = [
```

Comment:
`domains` has type `Option[List[String]]`.  So I think here you force the default to be `Some(Nil)`.  The problem is, I don't know to override this in the user-provided hocon to set it back to `None` again.  Is there a way?

I think a more sensible default is `None` i.e. not present in the hocon.